src/app/space/create/deployments/apps/deployment-card.component.spec.ts

```diff
@@ -60,10 +61,10 @@ describe('DeploymentCardComponent', () => {
     mockSvc = {
       getApplications: () => { throw 'Not Implemented'; },
```

Comment:
Would it be possible to use the test-mock utility here?

Reply:
I have a http followup patch in the works which already includes the use of `createMock` in several places, since modifying the DeploymentsService class by adding Http, AuthenticationService, etc. properties breaks those test suites' existing mocks. Is it okay if that's done in the followup instead?

Reply:
Okay that's fine with me.